lib/rubocop/cop/rails/validation.rb

```diff
@@ -48,6 +48,24 @@ def on_send(node)
         def preferred_method(method)
           WHITELIST[BLACKLIST.index(method.to_sym)]
         end
+
+        def autocorrect(node)
+          _receiver, method_name, *args = *node
+          options = args.delete_if { |arg| arg.type == :sym }.first
+          lambda do |corrector|
+            validate_parameter = method_name.to_s.split('_')[1]
```

Comment:
Didn't read the code carefully. Seems to me this should be named `validation_type` then (or something like this).